themes/beaver/assets/css/footer.css

```diff
@@ -0,0 +1,109 @@
+.footer-component {
+    background-color: black;
+    color: white;
+    padding-top: 130px;
+    padding-bottom: 40px;
+    font-size: medium;
+}
+
+.footer-component .logo {
+    width: 200px;
+    display: block;
+    margin-bottom: 16px;
+}
+
+.footer-component .footer-container {
+    max-width: 1280px;
+    margin: 0 auto;
+}
+
+.footer-component .flex-container {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+}
+
+.footer-component .headline {
+    margin-top: 0;
+    margin-bottom: 30px;
+    font-size: 18px;
+    font-weight: bold;
+}
+
+.footer-component ul {
+    list-style-type: none;
+    padding: 0;
+}
+
+.footer-component ul li {
+    margin: 12px 0;
+}
+
+.footer-component a {
+    color: white;
+    text-decoration: none;
+}
+
+.footer-component .flex-container .flex1 {
+    flex: 1
+}
+
+.footer-component .flex-container .flex2 {
+    flex: 2
+}
+
+.footer-component .flex-container .flex3 {
+    flex: 3
+}
+
+/* Large devices */
+@media (min-width: 992px) {
+    .footer-component .flex-container {
+        flex-direction: row;
+    }
+}
+
+
+
+.footer-component li a strong {
```

Comment:
It's beneficial to utilize classes rather than hard-coded HTML tags. This approach helps reduce coupling and mitigates potential issues when we need to modify the HTML structure in the future. For more information, you can refer to [Avoid tag selectors](https://ricostacruz.com/rscss/elements.html).

Reply:
@pftg fixed